src/main/scala/edu/cmu/cs/obsidian/Main.scala

```diff
@@ -346,9 +397,9 @@ object Main {
                 } catch {
                     case e: Throwable => println("Error running protoc: " + e)
                 }
+                generateFabricCode(mainName, options.outputPath, srcDir)
```

Comment:
Only in non-mock mode?

Reply:
Yep, that was during testing, good catch. Thanks.